### PR TITLE
OCPQE-26461 Add 4.15 autorelease jobs

### PR DESCRIPTION
**Pull request by jhuttana**
Add e2e/upgrade auto release jobs for 4.15 release

### CODE REVIEW

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.14-aws-ipi-f360

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f360

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f360

**Comment by rioliu-rh** [comment]
let's follow spec in PR https://github.com/openshift/release/pull/58213, change test frequency to f999.

**Comment by jhuttana** [comment]
> let's follow spec in PR #58213, change test frequency to f999.

Sure I will take a look at the changes there and modify this PR accordingly.

**Comment by jhuttana** [comment]
> > let's follow spec in PR #58213, change test frequency to f999.
> 
> Sure I will take a look at the changes there and modify this PR accordingly.

@rioliu-rh I have made the suggested changes to this PR . Could you please take a look?

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.14-aws-ipi-f999

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release-stable-4.15-upgrade-from-stable-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release-stable-4.15-upgrade-from-stable-4.14.yaml:42]: The `build_root` section is not needed for this yaml file, and also applies to the other config yaml files in this PR. 
Refer to https://github.com/openshift/release/pull/58218

**Comment by rioliu-rh** [comment]
> > let's follow spec in PR #58213, change test frequency to f999.
> 
> Sure I will take a look at the changes there and modify this PR accordingly.

cool, thanks, TIP: let's sync the job name with test registry, thanks

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release-stable-4.15-upgrade-from-stable-4.14.yaml (hunk 1 above)
Comment (jhuttana) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__automated-release-stable-4.15-upgrade-from-stable-4.14.yaml:42]: Sure. I will make those changes. Thank you!

**Comment by jhuttana** [comment]
> > > let's follow spec in PR #58213, change test frequency to f999.
> > 
> > 
> > Sure I will take a look at the changes there and modify this PR accordingly.
> 
> cool, thanks, TIP: let's sync the job name with test registry, thanks

Sorry I am not getting that @rioliu-rh.
Could you please point me to the sample reference how it should be done?

**Comment by jhuttana** [comment]
> > > let's follow spec in PR #58213, change test frequency to f999.
> > 
> > 
> > Sure I will take a look at the changes there and modify this PR accordingly.
> 
> cool, thanks, TIP: let's sync the job name with test registry, thanks

I have made the changes in PR https://github.com/openshift/release-tests/pull/334 to sync the job names. 
Could you please take a look?

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.14-aws-ipi-f999

**Comment by jhuttana** [comment]
> @jhuttana: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> Test name 	Commit 	Details 	Required 	Rerun command
> ci/rehearse/periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999 	[3d981eb](https://github.com/openshift/release/commit/3d981eb22401508afcb782159afd634b68f43672) 	[link](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58145/rehearse-58145-periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999/1851182186741370880) 	unknown 	`/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999`
> 
> [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openshift&repo=release&pr=58145). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:jhuttana).

I see similar failure in https://github.com/openshift/release/pull/58218. I am not sure whether it is fixed  or considered as known failure. 
@liangxia  and @rioliu-rh could you please clarify?

**Comment by rioliu-rh** [comment]
> > @jhuttana: The following test **failed**, say `/retest` to rerun all failed tests or `/retest-required` to rerun all mandatory failed tests:
> > Test name 	Commit 	Details 	Required 	Rerun command
> > ci/rehearse/periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999 	[3d981eb](https://github.com/openshift/release/commit/3d981eb22401508afcb782159afd634b68f43672) 	[link](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/58145/rehearse-58145-periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999/1851182186741370880) 	unknown 	`/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999`
> > [Full PR test history](https://prow.ci.openshift.org/pr-history?org=openshift&repo=release&pr=58145). [Your PR dashboard](https://prow.ci.openshift.org/pr?query=is:pr+state:open+author:jhuttana).
> 
> I see similar failure in #58218. I am not sure whether it is fixed or considered as known failure. @liangxia and @rioliu-rh could you please clarify?

I think the UI case failure is not related to this change.

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.15-aws-ipi-f999

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.14-azure-ipi-f999

**Comment by jhuttana** [comment]
@rioliu-rh  do we need to hold on merging this PR till UI related issue that is seen for prowjobs get fixed?

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.15-aws-ipi-f999

**Comment by jhuttana** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-stable-4.15-upgrade-from-stable-4.14-azure-ipi-f999

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-aws-ipi-f999

**Comment by rioliu-rh** [comment]
> Details

not a big deal, seems like upgrade jobs work fine. I triggered job run to verify e2e test

**Comment by rioliu-rh** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-automated-release-gcp-ipi-f999

**Pull request merged**